gcexport.py

```diff
@@ -903,12 +975,12 @@ def main(argv):
                 pass
                 # Display which entry we're skipping.
                 print('Skipping Garmin Connect activity ', end='')
-                print('(' + str(current_index) + '/' + str(total_to_download) + ') ', end='')
+                print('(' + str(current_index) + sep + str(total_to_download) + ') ', end='')
```

Comment:
```suggestion
                print(f'({current_index}{sep}{total_to_download}) ', end='')
```

Reply:
Python2 compatibility update
```suggestion
                print('(', current_index, sep, total_to_download, ') ', sep='', end='')
```